### DISCUSSION TITLE
Columns re-shuffled

### DISCUSSION THREAD

**Discussion by lykaodan** [Budibase/budibase#4029] [Help]
Hello! I connected my PostgreSQL database in my project, but the columns re-shuffled after added my database to budibase. How can I arrange them in order?

**Comment by mjashanks** [comment]
Do you mean that the columns in the `Data` section reordered? if so, there is no way to change this at the moment.

If you are referring to a `Table` component within your app, you can do this by selecting all the columns, in the order in which you need them.

![image](https://user-images.githubusercontent.com/3524181/149550951-3d4a5d83-cd3f-41fe-9a1e-fd66e9780eeb.png)
